### PR TITLE
Update parameter for scheduler.md

### PR DESCRIPTION
**Pull request by gyliu513**
Fix issue #18325

### CODE REVIEW

**Comment by karlkfi** [comment]
You, know, after looking at this for the umteenth time... The reason it's spelled wrong isn't because the docs are wrong, but because the flag itself is spelled wrong... The variable itself is spelled right: `ExplicitReconciliationMaxBackoff`.  There's no such thing as "reconciliantion"!

I'm 99% sure the correct fix is just to fix the flag. @jdef you got an opinion?

@jay-lau sorry for putting you through all this mess...

**Comment by jdef** [comment]
correct there is a misspelling in the .go code, e.g.
https://github.com/kubernetes/kubernetes/blob/b9cfab87e33ea649bdd13a1bd243c502d76e5d22/contrib/mesos/pkg/scheduler/config/config.go#L59

**Comment by jay-lau** [comment]
@jdef  so you mean we had better update the go code but not the md file, right?

File: contrib/mesos/docs/scheduler.md [hunk 1]
```diff
@@ -136,7 +136,6 @@ offer-ttl = 5s
 ; duration an expired offer lingers in history
 offer-linger-ttl = 2m
 
-<<<<<<< HEAD
```
Comment (jdef) [inline contrib/mesos/docs/scheduler.md]: :+1:

File: contrib/mesos/docs/scheduler.md [hunk 2]
```diff
@@ -151,10 +150,10 @@ framework-id-refresh-interval = 30s
 initial-implicit-reconciliation-delay = 15s
 
 ; interval in between internal task status checks/updates
-explicit-reconciliation-max-backoff = 2m
+explicit-reconciliantion-max-backoff = 2m
```
Comment (jdef) [inline contrib/mesos/docs/scheduler.md]: change the .go code instead

File: contrib/mesos/docs/scheduler.md [hunk 3]
```diff
@@ -151,10 +150,10 @@ framework-id-refresh-interval = 30s
...
 
-; waiting period after attempting to cancel an ongoing reconciliation
-explicit-reconciliation-abort-timeout = 30s
+; waiting period after attempting to cancel an ongoing reconciliantion
+explicit-reconciliantion-abort-timeout = 30s
```
Comment (jdef) [inline contrib/mesos/docs/scheduler.md]: change the .go code instead

**Comment by jdef** [comment]
I've clarified my intent be commenting on specific lines

File: contrib/mesos/docs/scheduler.md (hunk 1 above)
Comment (jay-lau) [inline contrib/mesos/docs/scheduler.md]: OK, will upload a new patch to address this conflict issue.

File: contrib/mesos/docs/scheduler.md (hunk 3 above)
Comment (jay-lau) [inline contrib/mesos/docs/scheduler.md]: ok, will create a new patch to handle this. ;-)

**Comment by jay-lau** [comment]
I was handling this in another two tickets https://github.com/kubernetes/kubernetes/pull/18574 and https://github.com/kubernetes/kubernetes/pull/18575

**Comment by karlkfi** [comment]
closing as duplicate

**Pull request closed without merging**